src/simdat.jl

```diff
@@ -15,10 +16,21 @@ Dict(
     "factor2_name" => ["F2_level1", "F2_level2", "F2_level3"]
 )
 """
```

Comment:
Not sure what is going on with the documentation string here.  This documentation string appears in the output from `makedocs` in the docs directory but not when using the REPL help command.

Reply:
I think I have fixed it.

Reply:
was the stub definition causing a problem? Weird. But then again stubs are usually only used for defining an API that you expect other packages to extend / implement. Since we actually have a method definition here, it doesn't matter.